docs/UserGuide.md

```diff
@@ -104,53 +158,85 @@ Examples:
 * `find Betsy Tim John`<br>
   Returns Any person having names `Betsy`, `Tim`, or `John`
 
-### 2.6. Deleting a person : `delete`
+### 2.5.2. Find most urgent tasks: `findurgent`
+
+Finds tasks that has the highest priority and closest deadline
+Format: `findurgent`
+
+### 2.5.3. Find all tags containing any keywords: `findtag`
 
-Deletes the specified person from the address book. Irreversible.<br>
+Finds tags whose name contain any of the given keywords.<br>
+Format: `findtag KEYWORD [MORE_KEYWORDS]`
```

Comment:
Once again, see if you can combine the different variants for find into the `find` command itself.